src/action-creators/newThoughtAtCursor.js

```diff
@@ -19,14 +19,18 @@ import {
   pathToContext,
   perma,
   splitChain,
-} from '../util'
+  isDivider
+} from '../util.js'
 
 export const newThoughtAtCursor = () => dispatch => {
 
   const { cursor, contextViews } = store.getState()
 
   let value = '' // eslint-disable-line fp/no-let
-  const offset = window.getSelection().focusOffset
+  const offset = isDivider(headValue(cursor))
+    ? headValue(cursor).length
+    : window.getSelection().focusOffset
```

Comment:
While this does prevent the issue from occurring, it doesn't go to the root of the problem. You have to ask, why is the offset wrong to begin with? It looks like the browser selection is still active when the divider is created. If you clear the browser selection when the divider is created, the problem is solved without any custom logic in `newThoughtAtCursor`. Fixed in e8a71d953df437646615257984b2988ce3612d33.